### DISCUSSION TITLE
Automatic refresh on opening the app

### DISCUSSION THREAD

**Discussion by nrestell** [breezy-weather/breezy-weather#502] [Q&A] [answered]
Hi,

Is it possible to configure the app to automatically refresh upon first opening? It is the first thing I do whenever I open the app to make sure the precipitation in the next 120 minute (Accuweather) is up-to-date.

Neil

**Comment by papjul** [comment, marked as answer]
Please [read FAQ](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md)
